bioblend/galaxy/tool_dependencies/__init__.py

```diff
@@ -0,0 +1,82 @@
+"""
+Contains possible interaction dealing with Galaxy dependency resolvers.
+"""
+from bioblend.galaxy.client import Client
+
+
+class ToolDependenciesClient(Client):
+
+    def __init__(self, galaxy_instance):
+        self.module = 'dependency_resolvers'
+        super().__init__(galaxy_instance)
+
+    def summarize_toolbox(self, index=None, tool_ids=None, resolver_type=None, include_containers=None, container_type=None, index_by=None):
```

Comment:
I'm not sure whether these should all default to None, e.g. `include_containers` can default to requirements.